cg/store/models.py

```diff
@@ -967,6 +968,7 @@ class RunDevice(Base):
 
     id: Mapped[PrimaryKeyInt]
     type: Mapped[DeviceType]
+    internal_id: Mapped[UniqueStr64]
```

Comment:
Would it make more sense to call this `device_name`?

Reply:
This is the name that was decided in the refinement. We wanted it to be an id because is unique and we wanted to mimic what it is used in the `Sample` model, so referring to it as an "internal_id" in the database models but as just `_id` in cg

Reply:
In the current context, name how the customers refer to the objects (like sample.name and case.name). I think internal_id makes sense in the reality we have. The alternative would be to refactor the other ones to sample.external_id or similar which I think is out of scope here